### ISSUE TITLE
javascript.com is just an ad for Pluralsight

### ISSUE THREAD

**Issue by ricealexander** [braziljs/js-the-right-way#330]
Under "Portals" in the "Reading" section, the very first link is for [JavaScript.com](https://javascript.com).

The site contains:
* Pluralsight branding and advertisements.
* A "free course" consisting of punching JavaScript commands into a terminal for under 5 minutes. (not updated for ES6)
* Links to Pluralsight's paid offerings
* A skill assessment that requires registering for Pluralsight
* Links to webinars that require registering for Pluralsight
* A handful of links to useful non-Pluralsight resources, under "Resources"

In other words, the entire site is a thinly veiled ad for Pluralsight and probably shouldn't be listed under Portals.

I would suggest removing this link, or at least titling it "Pluralsight" instead of "JavaScript.com".

Alternatively, we could replace it with [JavaScript.info](https://javascript.info) a legitimate, free, high-quality, up-to-date, and beginner-friendly reference.
